downloads/downloads-impl/src/main/java/com/duckduckgo/downloads/impl/DefaultFileDownloadNotificationManager.kt

```diff
@@ -52,7 +52,7 @@ private const val SUMMARY_ID = 0
 )
 @SingleInstanceIn(AppScope::class)
 class DefaultFileDownloadNotificationManager @Inject constructor(
-    private val notificationManager: NotificationManager,
+    private val notificationManager: NotificationManagerCompat,
```

Comment:
in `DeviceShieldAlertNotificationBuilder`, `NotificationModule` we use `NotificationManager` should we replace them there too?

Reply:
Good point! Added https://app.asana.com/0/0/1203771987668114/f to take care of it as a separate PR.